src/network/invthread.py

```diff
@@ -20,8 +41,9 @@ def __init__(self):
 
     def handleLocallyGenerated(self, stream, hashId):
         Dandelion().addHash(hashId, stream=stream)
-        for connection in BMConnectionPool().inboundConnections.values() + \
-            BMConnectionPool().outboundConnections.values():
+        for connection in \
+                BMConnectionPool().inboundConnections.values() + \
+                BMConnectionPool().outboundConnections.values():
                 if state.dandelion and connection != Dandelion().objectChildStem(hashId):
                     continue
                 connection.objectsNewToThem[hashId] = time()
```

Comment:
You used `with i.objectsNewToThemLock:` for similar expression above

Reply:
It's not necessary to lock here, because this is a new object not seen anywhere on the network. I think that's how it works.